README.md

```diff
@@ -1,2 +1,22 @@
-# website
-The website for TF Encrypted
+# TF Encrypted Website
+
+This repo contains the code for the [TF Encrypted](https://github.com/tf-encrypted/tf-encrypted) project.
```

Comment:
This repository contains the code for the TF Encrypted Website, we should link to it here! It'd also be good to describe what technologies are used and any dependencies required to run it.

Reply:
the name servers need to be updated .. technology is in the section below

Reply:
@ianlivingstone updated to mention next.js